jest.config.ts

```diff
@@ -1,14 +1,17 @@
-import type { InitialOptionsTsJest } from 'ts-jest/dist/types';
+import type { JestConfigWithTsJest } from "ts-jest";
 
-const config: InitialOptionsTsJest = {
-  preset: 'ts-jest',
-  testPathIgnorePatterns: ['dist'],
-  setupFilesAfterEnv: ['jest-extended/all', 'expect-more-jest'],
+const config: JestConfigWithTsJest = {
+  preset: "ts-jest",
```

Comment:
should be removed, otherwise js files are also transformed

Reply:
The preset?

Reply:
yes, as now the preset transform and our transform are merged. before the global option was overwritten.